client/my-sites/plugins/plugin-site/plugin-site.jsx

```diff
@@ -21,4 +24,24 @@ const PluginSite = ( props ) => {
 	return <PluginSiteJetpack { ...props } />;
 };
 
-export default PluginSite;
+export default connect(
+	( state, ownProps ) => {
+		const pluginSlug = get( ownProps, 'plugin.slug' );
+		const siteId = get( ownProps, 'site.ID' );
+
+		if ( includes( [ 'jetpack', 'vaultpress' ], pluginSlug ) && isSiteAutomatedTransfer( state, siteId ) ) {
+			return {
+				...ownProps,
+				...{
```

Comment:
Isn't this equivalent to:
```jsx
{
    ...ownProps,
    allowedActions: {
        activation: false,
        autoupdate: false,
        remove: false,
    },
    isAutoManaged: true,
}
```

Speaking of the data structure, is there any reason why we can't flatten this out? Open to name suggestions but something like  `siteCanActivatePlugin`, `siteCanAutoupdatePlugin`,  `siteCanRemovePlugin`